unity-renderer/Assets/Scripts/Tests/TestHelpers.cs

```diff
@@ -57,6 +57,13 @@ public static PB_Transform GetPBTransform(Vector3 position, Quaternion rotation,
             return pbTranf;
         }
 
+        public static FeatureFlag CreatetFeatureFlag()
+        {
+            FeatureFlag featureFlag = new FeatureFlag();
+            featureFlag.flags.Add("Tutorial", false);
```

Comment:
I think the feature names can be passed as a parameter instead of forcing "Tutorial". This can help to make more tests in the future.